lib/_tls_wrap.js

```diff
@@ -1056,18 +1056,14 @@ exports.connect = function(...args /* [port,] [host,] [options,] [cb] */) {
     socket.once('secureConnect', cb);
 
   if (!options.socket) {
-    var connect_opt;
-    if (options.path && !options.port) {
-      connect_opt = { path: options.path };
-    } else {
-      connect_opt = {
-        port: options.port,
-        host: options.host,
-        family: options.family,
-        localAddress: options.localAddress,
-        lookup: options.lookup
-      };
-    }
+    const connect_opt = {
```

Comment:
Nit: I know that this uses the same variable name as the old one, but most part of the core uses camel case. Could you change this?